src/Symfony/Component/DependencyInjection/ParameterBag/ParameterResolverInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\ParameterBag;
+
+use Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException;
+use Symfony\Component\DependencyInjection\Exception\ParameterCircularReferenceException;
+use Symfony\Component\DependencyInjection\Exception\RuntimeException;
+
+/**
+ * Replaces parameter placeholders (%name%) by their values.
+ *
+ * @author Guilhem N. <egetick@gmail.com>
+ */
+interface ParameterResolverInterface
```

Comment:
I was thinking using it for other classes than `ParameterBag` but indeed we can remove it unless someone explicitly requests it.